_posts/2019-06-05-materials.md

```diff
@@ -72,33 +81,20 @@ address="Cham",
 
 > SROS1 is a proposed addition to the ROS1 API and ecosystem to support modern cryptography and security measures. An overview of current progress will be presented, explaining each major advancement, including: over-the-wire cryptography for all data transport, namespaced access control enforcing graph policies/restrictions, and finally process profiles using Linux Security Modules to harden a node's resource access. This chapter not only seeks to raise community awareness of the vulnerabilities in ROS1, but to provide clear instruction along designed patterns of development for using proposed solutions provided by SROS1 to advance the state of security for open source robotics subsystems.
 
-<pre>
-  <code class="ruby">
-@Inbook{White2019SROS1,
-author="White, Ruffin and Caiazza, Gianluca and Christensen, Henrik and Cortesi, Agostino",
-editor="Koubaa, Anis",
-title="SROS1: Using and Developing Secure ROS1 Systems",
-bookTitle="Robot Operating System (ROS): The Complete Reference (Volume 3)",
-year="2019",
-publisher="Springer International Publishing",
-address="Cham",
-pages="373--405",
-isbn="978-3-319-91590-6",
-doi="10.1007/978-3-319-91590-6_11",
-url="https://doi.org/10.1007/978-3-319-91590-6_11"
-}
-</code>
-</pre>
```

Comment:
This has been removed for convenience of block commenting but can be easily added back once we sync on what content and references to list on the website